contracts/azorius/LinearERC20Voting.sol

```diff
@@ -153,17 +171,13 @@ contract LinearERC20Voting is BaseStrategy, BaseQuorumPercent {
 
     /** @inheritdoc IBaseStrategy*/
     function isPassed(uint256 _proposalId) public view override returns (bool) {
-        if (
-            proposalVotes[_proposalId].yesVotes > proposalVotes[_proposalId].noVotes &&
-            proposalVotes[_proposalId].yesVotes >=
-            quorum(proposalVotes[_proposalId].votingStartBlock) &&
-            proposalVotes[_proposalId].votingEndBlock != 0 &&
-            block.number > proposalVotes[_proposalId].votingEndBlock
-        ) {
-            return true;
-        }
-
-        return false;
+        return (
+            proposalVotes[_proposalId].votingEndBlock != 0 && // end block wasn't set to 0 TODO why do we need this?
+            block.number > proposalVotes[_proposalId].votingEndBlock && // voting period has ended
+            proposalVotes[_proposalId].yesVotes >= quorum(proposalVotes[_proposalId].votingStartBlock) && // yes votes meets the quorum
+            block.number > proposalVotes[_proposalId].votingEndBlock && // more yes votes than no (simple majority)
+            proposalVotes[_proposalId].yesVotes / (proposalVotes[_proposalId].yesVotes + proposalVotes[_proposalId].noVotes) > basis() // yes votes meets the basis
```

Comment:
This math also won't work for the same reason as above. Will probably need to be something like: 

`yesVotes > (yesVotes + noVotes) * basisNumerator / basisDenominator`